web/templates/twig-source/incuna-surveys/form/base/proportion-field.html

```diff
@@ -20,11 +12,15 @@
                         >
                     {% block input %}
                         <div class="proportion-input" integer-field model="model[$index]" id="field.id" form="form"></div>
+                        <div
+                            aif-slider-input
+                            model="model[$index]"
+                            ceiling="100"
```

Comment:
I think the `width` would need to just be standard `css` asn would probably want to be on a containing div - I think someone from @incuna/frontend could look ta it but they would need the `remaining` value in the scope - possibly using a simple directive to set the `model[$index] + remaining` value.